src/app/providers/device-authentication/__tests__/device-authentication.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { TestBed } from '@angular/core/testing';
+import { Platform } from '@ionic/angular';
+import { configureTestSuite } from 'ng-bullet';
+import { DeviceAuthenticationProvider } from '../device-authentication';
+import { AppConfigProvider } from '../../app-config/app-config';
+import { AppConfigProviderMock } from '../../app-config/__mocks__/app-config.mock';
+import { AppConfig } from '../../app-config/app-config.model';
+
+describe('Device Authentication Provider', () => {
+
+  let deviceAuthenticationProvider: DeviceAuthenticationProvider;
+  const platformMock = {
```

Comment:
can we centralise this into a PlatformMockClass under the assumption this will probably be needed a few times?